Firmware/mmu2.cpp

```diff
@@ -549,6 +555,7 @@ void MMU2::manage_response(const bool move_axes, const bool turn_off_nozzle) {
             st_synchronize(); 
             return;
         case VersionMismatch: // this basically means the MMU will be disabled until reconnected
+            CheckUserInput();
```

Comment:
At this moment the only thing we can do is disabling the MMU. In the future we'll be able to flash the new FW as well thanks to @leptun